.nvmrc

```diff
@@ -0,0 +1 @@
+v17.9.0
```

Comment:
`yarn` will not run with this version

Reply:
My yarn runs with this version.
But like we discussed: I put it in ignore. @Mogge